source/includes/v4/_tickets.md

```diff
@@ -374,3 +374,57 @@ Note that the ticket response block may not contain all of your requested ticket
 ### Reservations
 
 Provide an array of reservation tokens to be considered in calculation
+
+## Contents for a set of tickets
+
```

Comment:
Please add a short description, what the Endpoint does:

"This call returns dynamic content for a set of tickets. Use this to optimise content fetching."